controllers/reconcilers/kserve_isvc_service_cert_reconciler.go

```diff
@@ -0,0 +1,132 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package reconcilers
+
+import (
+	"context"
+	"time"
+
+	"github.com/go-logr/logr"
+	kservev1beta1 "github.com/kserve/kserve/pkg/apis/serving/v1beta1"
+	"github.com/opendatahub-io/odh-model-controller/controllers/constants"
+	"github.com/opendatahub-io/odh-model-controller/controllers/resources"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var _ SubResourceReconciler = (*KserveIsvcServiceReconciler)(nil)
+
+type KserveIsvcServiceReconciler struct {
+	client         client.Client
+	serviceHandler resources.ServiceHandler
+}
+
+func NewKserveIsvcServiceReconciler(client client.Client) *KserveIsvcServiceReconciler {
+	return &KserveIsvcServiceReconciler{
+		client:         client,
+		serviceHandler: resources.NewServiceHandler(client),
+	}
+}
+
+func (r *KserveIsvcServiceReconciler) Delete(ctx context.Context, log logr.Logger, isvc *kservev1beta1.InferenceService) error {
+	// NOOP - Resources are deleted along with the deletion of InferenceServices
```

Comment:
same here, who is setting the ownership?

Reply:
This is managed by kserve.